starter-code/json-hw.js

```diff
@@ -1,32 +1,59 @@
-/*
-1. Convert the string `jsonString` to a Javascript Object and store it in a variable
-called `flickerObj`
-*/
 
+// 1. Convert the string `jsonString` to a Javascript Object and store it in a variable
 
+// called `flickerObj`
 
+var flickerObj = JSON.parse(jsonString);
 
 /*
+
 2. Iterate (this means 'use a loop') and do a console.log of each of the item titles
+
 with the date the photo was taken. The first console.log should return the following:
 
 "What a surprise in lyon. 2016-02-23T06:47:30-08:00"
 
 */
 
+//iterate through items and display title and date_taken on the console
+
+for (var i = 0; i < flickerObj.items.length; i++) {
+
+//display details
+
+console.log(flickerObj.items[i].title + "." + flickerObj.items[0].date_taken);
+
+}
 
 /*
+
 3. Create a new array called 'links'. Iterate through the items array in flickerObj and store the the links to each item in the 'links' array.
+
 */
 
+var links = [];//array
 
+for (var i = 0; i < flickerObj.items.length; i++) {
+
+//store link into links array
+
+links[i] = flickerObj.items[i].link;
```

Comment:
Here you can also use `push`
```suggestion
links.push(flickerObj.items[i].link);
```